spec/lib/archive_importer_spec.rb

```diff
@@ -155,6 +156,7 @@
       expect(archive_importer.user.language).to eq("ru")
       expect(archive_importer.user.disable_mail).to eq(false)
       expect(archive_importer.user.auto_follow_back).to eq(true)
+      expect(archive_importer.user.getting_started).to eq(archive_hash['user']['getting_started'])
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.